ingest/expression_files/expression_files.py

```diff
@@ -138,6 +156,59 @@ def check_unique_cells(cell_names: List, study_id, client):
             raise ValueError(error_string)
         return True
 
+    @staticmethod
+    def get_raw_count_query_filters(
+        study_file_id, raw_count_study_file_ids: List[Dict], include_study_file_id=True
+    ) -> List[Dict]:
+        """Creates query filters for study files that are considered raw count matrices.
+
+            Every result in 'raw_id_query_results' is expected to have the id present. This
+                function can eventually expand to generically create filters for a given property.
+
+        :parameter
+            study_file_id (ObjectID) : Study file id for the current study
+            raw_count_study_file_ids : Study file ids that need filters created ie. [{'id': <Study file id>}]
+            include_study_file_id : Whether to include 'study_file_id' in filter
+        """
+
+        def generate_filters(raw_count_study_file_ids):
+            return [
+                {"study_file_id": result["_id"]} for result in raw_count_study_file_ids
+            ]
+
+        current_study_file_id = {"_id": ObjectId(study_file_id)}
+        if not include_study_file_id:
+            if (item == current_study_file_id.id for item in raw_count_study_file_ids):
+                raw_count_study_file_ids.remove(current_study_file_id)
+                # raw_count_study_file_ids has other study file
+                if raw_count_study_file_ids:
+                    # Builds study ids fields for query
+                    return generate_filters(raw_count_study_file_ids)
+                else:
+                    return None
+        else:
+            return generate_filters(raw_count_study_file_ids)
+
+    @staticmethod
+    def get_raw_count_study_file_ids(client, study_id) -> List[Dict]:
+        """Returns raw count study file ids that are apart of study"""
+
+        COLLECTION_NAME = "study_files"
+
+        raw_ids_query = {
+            "$and": [
+                {"study_id": study_id},
+                {"file_type": "Expression Matrix"},
```

Comment:
Blocker: this entails only raw dense matrices are supported.  We also need to support raw MTX matrices.